### PR TITLE
Removed separate multicity_input Animations.

### PR DESCRIPTION
**Pull request by d3xvn**
Removed separate multicity_input Animations.

### CODE REVIEW

**Comment by MarcinusX** [comment]
Hey man! Thanks for PR! It looks much better!
I just have one question: Do you know if it's ok to create those animations inside the build method instead of having them stored in State object?

**Comment by d3xvn** [comment]
I think it's okay considering we're not rerunning the animation or even rebuilding the layout using setState(). My first instinct when I read your question was, if we run setState() at any point during or after the animation, it might restart. But when I did it myself using Future.delayed(), it didn't rerun either time. So I think it's fine compared to a state object.

File: lib/multicity_input.dart [hunk 1]
```diff
@@ -11,36 +11,13 @@ class MulticityInput extends StatefulWidget {
...
-    passengersAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.45, 0.65, curve: Curves.linear));
-    departureAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.6, 0.8, curve: Curves.linear));
-    arrivalAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.75, 0.95, curve: Curves.linear));
+    textInputAnimationController.forward();
```
Comment (MarcinusX) [inline lib/multicity_input.dart]: It's not supposed to run on init. Only on plus click. Could you remove this line and then I will merge, ok? :)

Comment (d3xvn) [reply lib/multicity_input.dart]: I guess I just forgot to remove that line after testing, I'll do it as soon as I can!

Comment (d3xvn) [reply lib/multicity_input.dart]: I've removed line 20 starting the animation. Sorry for the delay.

**Review by MarcinusX — APPROVED** [review]

**Comment by MarcinusX** [comment]
Thank you for the contribution!

**Pull request merged**